client/components/risk-level/index.tsx

```diff
@@ -0,0 +1,42 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import riskMappings from './strings';
+import Pill from '../pill';
+
+type RiskLevelNames = 'normal' | 'elevated' | 'highest' | 'not_assessed';
```

Comment:
@Automattic/pulsar, any opinions on keeping the definitions here vs keeping them in under types definitions in  'client/types/' folder?

Reply:
I was divided here, but then saw other components having types in the same file, where number of types were fewer and with less complex interfaces. [Example](https://github.com/Automattic/woocommerce-payments//blob/b6e37ebf103aba3dc9be58daf9be647fcee14ce8/client/components/deposit-status-pill/index.tsx)

I will be happy to comply what we decide as the best practice. 👍